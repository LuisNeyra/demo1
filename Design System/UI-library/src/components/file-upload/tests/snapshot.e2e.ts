import { snapIt } from '../../../test';

const component = 'file-upload';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap(
      'correctly',
      `<blaze-file-upload>
        <button class="c-button c-button--info">Upload files</button>
      </blaze-file-upload>`
    );

    snap(
      'dropzone correctly',
      `<blaze-file-upload drop multiple accept=".md">
        Drop to upload your files
      </blaze-file-upload>`
    );
  });
});
