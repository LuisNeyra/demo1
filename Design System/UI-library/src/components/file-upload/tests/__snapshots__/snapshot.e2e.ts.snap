// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`file-upload renders correctly 1`] = `"<blaze-file-upload class=\\"hydrated\\"><!----><div class=\\"c-file-upload\\"> <button class=\\"c-button c-button--info\\">Upload files</button> <input type=\\"file\\"></div></blaze-file-upload>"`;

exports[`file-upload renders dropzone correctly 1`] = `"<blaze-file-upload drop=\\"\\" multiple=\\"\\" accept=\\".md\\" class=\\"hydrated\\"><!----><div class=\\"c-file-upload c-file-upload--drop\\"> Drop to upload your files <input type=\\"file\\" multiple=\\"\\"></div></blaze-file-upload>"`;
